pom.xml

```diff
@@ -11,6 +11,7 @@
         <module>sample-use</module>
         <module>imageprocessor</module>
         <module>eventbus</module>
+        <module>imageprocessor-api</module>
```

Comment:
Wrong commit name, it's too general. You created two model class and create new module.